src/ae_kqueue.c

```diff
@@ -37,15 +37,21 @@ typedef struct aeApiState {
     int kqfd;
     struct kevent *events;
 
+    /* changes is used to buffer incoming events that will be
+     * registered in bulk via kevent(2). */
+    struct kevent changes[1024];
```

Comment:
```suggestion
    struct kevent changes[MAX_QUEUED_EVENTS];
```
Would rather this be an independent constant.